controllers/managedClusterResources.go

```diff
@@ -552,23 +553,24 @@ func (r *ClusterGroupUpgradeReconciler) getPrecacheJobTemplateData(
 	return rv, nil
 }
 
-func (r *ClusterGroupUpgradeReconciler) getBackupJobTemplateData(clusterGroupUpgrade *ranv1alpha1.ClusterGroupUpgrade, clusterName string) *templateData {
+// getBackupJobTemplateData initializes template data for the backup job creation
+// returns: 	*templateData
+//				error
+func (r *ClusterGroupUpgradeReconciler) getBackupJobTemplateData(clusterGroupUpgrade *ranv1alpha1.ClusterGroupUpgrade, clusterName string) (*templateData, error) {
 
 	rv := new(templateData)
-
 	rv.Cluster = clusterName
 	rv.JobTimeout = uint64(
 		clusterGroupUpgrade.Spec.RemediationStrategy.Timeout)
 
-	// TODO: currently using hard-coded recovery image, we must get
-	// image from csv.
-	//	image, err := r.getPrecacheimagePullSpec(ctx, clusterGroupUpgrade)
-	//	if err != nil {
-	//		return rv, err
-	//	}
-	//	rv.WorkloadImage = image
```

Comment:
This is going to fail the lint, since commented-out code is not allowed

Reply:
This is strange, the commented out code doesn't exist in my local branch! Need to investigate